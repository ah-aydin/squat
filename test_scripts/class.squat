class Person {
    string name;
    int age;
    int height;
}

func func1() {
    println("This is func 1");
}

func main() {
    Person person = Person("gigel", 20, 184);
    println(Person);
    println(person);
}
