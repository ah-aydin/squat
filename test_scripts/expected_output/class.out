Compiling and running file: test_scripts/class.squat
<class Person>
<instance of Person [String("gigel"), Int(20), Int(184)]>
gigel
The sum of the age and height is 204
204
<type Int>
Both people have in total 41 years lived
